change/@ni-nimble-components-56124785-d76b-4bd0-b330-b457eb4ff9a0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc looks ready, can you double check the PR description is accurate?

Reply:
Yep, now up-to-date.